AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -852,6 +986,7 @@ def test_FBX_ModifiedFBXFile_ConsistentProductOutput_OutputSucceeds(self, worksp
             5. Run asset processor
             6. Validate that Asset Processor generates the expected output
         """
+        import shutil
```

Comment:
Again, avoid imports in methods unless you have a very specific reason

Reply:
Fixed